### DISCUSSION TITLE
GitHub citation repository takes too long to load

### DISCUSSION THREAD

**Discussion by NEGU93** [community/community#34068] [Other Feature Feedback, Questions, & Ideas]
GitHub added a cool citation feature as documented [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files). 

I created my `citation.cff` in [this](https://github.com/NEGU93/cvnn) repository. However, it takes ages to load. If you click on the `cite this repository` button at the right it will take ages and, in my case, never waited long enough for it to load (so I don't know if it eventually does).

When I clone the repository, the `citation.cff` file is only `1KB` of size and this is the content:

``` cff
cff-version: 1.0
message: "If you use this software, please cite it as below."
authors:
  - family-names: BARRACHINA
    given-names: Jose Agustin
    orcid: https://orcid.org/0000-0002-2139-514X
title: "Complex-Valued Neural Networks (CVNN)"
version: 1.0.3
doi: https://doi.org/10.5281/zenodo.4452131
date-released: 2021-01
```

I am not sure if this is a problem with the content of the file or some issue with the server (like a corrupted file). What can I do to solve this?

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️

**Reply by NEGU93** [reply]
Hello,

Thank you for the reply.

I tried with two different computers on different days and internet connections. I don't think the problem is on my side.

Could you see it correctly? Did it load quickly?
That will help me know if it's a problem on my side.

Regards,

**Reply by 3kh0** [reply]
Hello @NEGU93, for me it took about 5 seconds to load but it did load.
